templates/admin/amp-status.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * AMP status option in the submit meta box.
+ *
+ * @package AMP
+ */
+
+// Check referrer.
+if ( ! ( $this instanceof AMP_Post_Meta_Box ) ) {
+	return;
+}
+?>
+<div class="misc-pub-section misc-amp-status">
+	<i></i>
```

Comment:
Why an `i` here and not a `span`?

Reply:
I guess there have been a debate for a while as "what should be used for icons". Font awesome, Facebook etc. exclusively use the `i` tag for icons for example. I am personally in favor of using the `i` tag because it adds a semantic meaning (even though it doesn't mean icon per se) while `span` doesn't.
That said, the various places in the WP admin which are not using the dashicon font use the `span` tag so I think it would be better to keep it as consistent as possible. [This commit](https://github.com/Automattic/amp-wp/pull/813/commits/e4f6c8733ebadf0766242a7c6e39e04393ca15e2) changes the `i` tag to a `span`.